src/test/java/JsonParserTests.java

```diff
@@ -0,0 +1,61 @@
+import org.junit.*;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import parser.Parser;
+import shop.Cart;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+
+
+public class JsonParserTests {
+
+    private Cart cart;
+    private File file;
+    private final String cartName = "andrew-cart2";
+    public static final String EXPECTED_CART_RESULT ="{\"cartName\":\"andrew-cart2\",\"realItems\":[],\"virtualItems\":[],\"total\":0.0}";
+
+
+    @Before
+    public void beforeTests() {
+        file = new File("src/main/resources/" + cartName + ".json");
+    }
+
+    @Disabled("Disabled")
+    @Test
+    public void readFromFileTest() {
+        Parser parser = new JsonParser();
```

Comment:
move the logic common to all tests to @BeforeEach method

Reply:
Fixed